### ISSUE TITLE
Getting it to work in Windows build/ getting micro:bit code

### ISSUE THREAD

**Issue by msRobot123** [bLiGM/Microbit-Unity#4]
So works great in editor, but the microbit doesn't seem to connect when I create a windows build. Does anybody have any advice on this? Also, is it at all possible to get the micropython/javascript code from the hex? I want to add and change some stuff.

**Comment by beatsaway** [comment]
me too... agree totally

**Comment by renebohne** [comment]
yes, please. Would be super helpful to have the micro:bit source code.
